ebuild-writing/variables/text.xml

```diff
@@ -614,8 +632,8 @@ to read.
 <p>
 Some ebuilds use calls to <c>sed</c>, <c>awk</c> and / or <c>cut</c> to do this. This
 must <e>not</e> be done for any new code, and should be fixed to use either
-eapi7-ver or bash substitution where possible. Global scope non-bash code is
-highly discouraged.
+<c>eapi7-ver</c> (before EAPI 7) or Bash substitution where possible. Global scope
+non-Bash code is <b>strongly</b> discouraged.
```

Comment:
I think this is misleading. The possibilities are a) Bash substitution for the cases where it is possible, b) version manipulation commands in EAPI 7, and c) eapi7-ver in older EAPIs.